.github/workflows/release.yaml

```diff
@@ -123,6 +127,55 @@ jobs:
       - name :  docker images publish
         run: |
           make docker-publish-cli
+      
+      - name: Make Release
+        env:
+          VERSION: ${{ steps.version.outputs.version }}
+        run: |
+          rm -rf release
+          make release
+          make release-notes > release/release-notes.out
+          cat release/release-notes.out
+          
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: latest
+          args: release --rm-dist --debug --release-notes=release/release-notes.out
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+  create-release:
```

Comment:
This action, it doesn't appear to actually do anything besides run go report card. I wonder if the `Make release` and `Run GoReleaser` (and `Set version`) should be moved under `create-release` workflow.